go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ava-labs/timestampvm
 go 1.16
 
 require (
-	github.com/ava-labs/avalanchego v1.6.2
+	github.com/ava-labs/avalanchego v1.6.5
```

Comment:
need to run `go mod tidy` to bump versions in the `go.sum`

Reply:
go.sum should now be updated